src/View.elm

```diff
@@ -0,0 +1,45 @@
+module View exposing (..)
+
+import Html exposing (..)
+import Models exposing (Model)
+import Messages exposing (Msg(..))
+import Routing.Route exposing (Route(..))
+import Landing.View as Landing
+import App.View as App
+
+
+view : Model -> Html Msg
+view model =
+    div []
+        [ page model ]
+
+
+page : Model -> Html Msg
+page model =
+    case model.route of
+        HomeRoute ->
```

Comment:
These messages are being mapped to the same view. Technically, this repetition would have been easier to remove if we could have a pattern match such as
```elm
case model.route of
    (HomeRoute, AboutRoute, FaqRoute) as landingRoutes -> ...
```
but Elm won't let me do that. 😓 

Or if the routes could be a union type of `LandingPageRoutes` and `AppRoutes`, for example.
```elm
type LandingPageRoutes = HomeRoute | AboutRoute | ...

type AppRoutes = BrownbagRoute | HangoutsRoute | ...

type Routes = LandingPageRoutes | AppRoutes
```
We could then pattern match against the incoming route and respond accordingly, theoretically. 😅 
Tried that but couldn't figure the `matchers` function in `Routing.Route` module.

Resorted to thinking  ahead, when we have something like `model.isAuthenticated`? Then we can use a simple `if-then` to render the `LandingPage` if the user is not authenticated, the `App` pages otherwise. Hence the current duplication and same view mapping.